includes/ProductsApi.php

```diff
@@ -57,31 +59,31 @@ public function register_routes() {
 	/**
 	 * Get products data
 	 *
-	 * @return \WP_Error|\WP_HTTP_Response|\WP_REST_Response
+	 * @return WP_Error|WP_HTTP_Response|WP_REST_Response
 	 */
 	public function products_callback() {
 
 		$products       = array();
 		$args           = array(
 			'method' => 'GET',
 		);
-		$hiive_response = $this->hiive->hiive_request( HIIVE_API_PRODUCTS_ENDPOINT, array(), $args );
+		$hiive_response = $this->hiive->hiive_request( self::HIIVE_API_PRODUCTS_ENDPOINT, array(), $args );
 
 		if ( is_wp_error( $hiive_response ) ) {
-			return new \WP_REST_Response( $hiive_response->get_error_message(), 401 );
+			return new WP_REST_Response( $hiive_response->get_error_message(), 500 );
```

Comment:
@mr-vara I changed this here because it's only sometimes 401. I think 500 is a more generic error code. I presume the JS client probably doesn't care about the code, just was it success/failure – is this change OK?